src/epub/text/those-in-peril-on-the-tee.xhtml

```diff
@@ -248,12 +248,12 @@
 			<p>“Exactly. Exactly. I quite understand. I have lost the match. So be it.”</p>
 			<p>And, with drooping shoulders, Frederick Pilcher shuffled off in the direction of the bar.</p>
 			<p>John Gooch watched him go with a seething fury which for the moment robbed him of speech. He might, he told himself, have expected something like this. Frederick Pilcher, lost to every sense of good feeling and fair play, had double-crossed him. He shuddered as he realized how inky must be the hue of Frederick Pilcher’s soul; and he wished in a frenzy of regret that he had thought of picking his own ball up. Too late! Too late!</p>
-			<p>For an instant the world had been blotted out for John Gooch by a sort of red mist. This mist clearing, he now saw Agnes Flack standing looking at him in a speculative sort of way, an odd expression in her eyes. And beyond her, leaning darkly against the club-house wall, his bulging muscles swelling beneath his coat and his powerful fingers tearing to pieces what appeared to be a section of lead piping, stood Sidney McMurdo.</p>
+			<p>For an instant the world had been blotted out for John Gooch by a sort of red mist. This mist clearing, he now saw Agnes Flack standing looking at him in a speculative sort of way, an odd expression in her eyes. And beyond her, leaning darkly against the clubhouse wall, his bulging muscles swelling beneath his coat and his powerful fingers tearing to pieces what appeared to be a section of lead piping, stood Sidney McMurdo.</p>
 			<p>John Gooch did not hesitate. Although McMurdo was some distance away, he could see him quite clearly; and with equal clearness he could remember every detail of that recent interview with him. He drew a step nearer to Agnes Flack, and, having gulped once or twice, began to speak.</p>
 			<p>“Agnes,” he said, huskily, “there is something I want to say to you. Oh, Agnes, have you not guessed⁠—”</p>
 			<p>“One moment,” said Agnes Flack. “If you’re trying to propose to me, sign off. There is nothing doing. The idea is all wet.”</p>
 			<p>“All wet?”</p>
-			<p>“All absolutely wet. I admit that there was a time when I toyed with the idea of marrying a man with brains, but there are limits. I wouldn’t marry a man who played golf as badly as you do if he were the last man in the world. Sid-nee!” she roared, turning and cupping her mouth with her hands; and a nervous golfer down by the lake-hole leaped three feet and got his mashie entangled between his legs.</p>
+			<p>“All absolutely wet. I admit that there was a time when I toyed with the idea of marrying a man with brains, but there are limits. I wouldn’t marry a man who played golf as badly as you do if he were the last man in the world. Sid-née!” she roared, turning and cupping her mouth with her hands; and a nervous golfer down by the lake-hole leaped three feet and got his mashie entangled between his legs.</p>
```

Comment:
Wrong change

Reply:
? She is yelling Sidney in an elongated fashion... no need for an accent.